app/Website/resources/views/mail/contact.blade.php

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
There is not stying in this e-mail. There is an option for markdown e-mail with style and some components already provided if you want to use that: https://laravel.com/docs/11.x/mail#writing-markdown-messages